apps/mooc/backend/src/Controller/VideoLike/VideoLikePostController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CodelyTv\Apps\Mooc\Backend\Controller\VideoLike;
+
+use CodelyTv\Mooc\VideoLike\Application\Create\CreateVideoLikeCommand;
+use CodelyTv\Shared\Domain\ValueObject\Uuid;
+use CodelyTv\Shared\Infrastructure\Symfony\ApiController;
+use CodelyTv\Shared\Infrastructure\Symfony\ApiHttpCreatedResponse;
+use Faker\Provider\Uuid as UuidFactory;
+use Symfony\Component\HttpFoundation\Request;
+
+final class VideoLikePostController extends ApiController
+{
+    protected function exceptions(): array
+    {
+        return [];
+    }
+
+    public function __invoke(Request $request)
+    {
+        /** @var CreateVideoLikeCommand */
+        $command = new CreateVideoLikeCommand(
+            (new Uuid(UuidFactory::uuid())),
```

Comment:
This id could be sent from the client. With that this controller could be a PUT one.